docs/paper/dev/getting-started/project-setup.mdx

```diff
@@ -187,6 +187,49 @@ If everything went well, you should see something like this:
 
 ![](assets/paper-plugin-overview.png)
 
+## Plugin remapping
+
+As of 1.20.5, Paper will be shipping with a Mojang-mapped runtime instead of reobfuscating the server to Spigot mappings.
```

Comment:
ships with, same in the other two places

Reply:
I just took jmp's message as reference. Will change it. Thank you for the feedback.